spec/github_helper_spec.rb

```diff
@@ -2,6 +2,30 @@
 require 'webmock/rspec'
 
 describe Fastlane::Helper::GithubHelper do
+  describe '#initialize' do
+    let(:client) do
+      instance_double(
+        Octokit::Client,
+        user: instance_double('User', name: 'test'),
+        'auto_paginate=': nil
+      )
+    end
+
+    before do
+      allow(Octokit::Client).to receive(:new).and_return(client)
+    end
+
+    it 'with the correct github_token' do
+      expect(described_class).to receive(:new).with(github_token: 'GITHUB_TOKEN')
+      described_class.new(github_token: 'GITHUB_TOKEN')
+    end
+
+    it 'Octokit client receives the correct github_token' do
```

Comment:
By convention, the name of the "examples" (that's what RSpec calls each `it` test in their unit test suites) are supposed to read as English sentences — that's why they use the keyword `it` to introduce those (as in 'it does x y z').

So I suggest something more like `it 'properly passes the token all the way down to the Octokit::Client'` or something in those lines.

Reply:
Thanks for the advice! I'll change it and review my other "Its" to proper align with the convention.